static/stable/stage/navigation/rhel-navigation.json

```diff
@@ -3,9 +3,9 @@
     "title": "Red Hat Insights",
     "navItems": [
       {
-        "id": "overview",
+        "id": "dashboard",
```

Comment:
This changes has to be reflected in `service.json` if the link is supposed to show up in the new dropdown

Reply:
Thank you! Do you mean all changes or only the change of nav id?

Reply:
Went through service.json under /stage/services folder. 

1. As the dashboard is not a real service but an overall page for RHEL insights, it seems we do not have to make any changes for Overview => dashboard change 
2. The inventory is renamed already inside the dropdown without changing it's ID. This also seems not needed to be reflected in /stage/services/servises.json file.

Let me know if I miss anything, it is my first PR in this new repo :)